### PR TITLE
Update file1

### PR DESCRIPTION
**Pull request by chitu1629**
Kindly review my code.

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Zomato is used to buy food through online.
 This is a app which has both veg and non-veg foods.
```
Comment (VeluPrabhakaranV) [inline file1]: Add upper case for veg and non-veg.

**Review by chitu1629 — COMMENTED** [review]
I have changed it.

**Comment by VeluPrabhakaranV** [comment]
It looks good.

**Comment by chitu1629** [comment]
Thanks for reviewing it.

**Review by VeluPrabhakaranV — APPROVED** [review]
Approving your code.

**Pull request merged**